test/apps/iodemo/io_demo.cc

```diff
@@ -87,6 +87,7 @@ typedef struct {
     bool                     use_epoll;
     ucs_memory_type_t        memory_type;
     unsigned                 progress_count;
+    const char*              local_addr;
```

Comment:
Rename all local address to source address

Reply:
renamed